src/Simplex/Messaging/Agent.hs

```diff
@@ -423,6 +577,7 @@ subscribeConnection' c connId =
         _ -> throwError $ INTERNAL "unexpected queue status"
     SomeConn _ (RcvConnection _ rq) -> subscribe rq
     SomeConn _ (ContactConnection _ rq) -> subscribe rq
+    SomeConn _ (NewConnection _) -> pure ()
```

Comment:
this is where command processing should be started for a given connection...

That probably doesn't fit data model - while we don't need separate queues per connection, we do need to know which connections are subscribed and only query commands for those connections...

Alternatively we may need to have separate queues, like we have for messages...